packages/react/src/create-strict-api/index.ts

```diff
@@ -146,7 +154,13 @@ interface CompiledAPI<TSchema extends CompiledSchema> {
       requiredProperties: TAllowedProperties;
       requiredPseudos: TAllowedPseudos;
     } = never
-  >(): Internal$XCSSProp<TAllowedProperties, TAllowedPseudos, TSchema, TRequiredProperties>;
+  >(): Internal$XCSSProp<
```

Comment:
Would have to touch `StrictXCSSProp` 👀

Reply:
Would you though?

Reply:
Nope, sorry, I didn't even look… `XCSSProp<TAllowedProperties, TAllowedPseudos, TRequiredProperties>` is untouched.